### DISCUSSION TITLE
新版的全局设置-DNS设置中的高级设置-用局域网DNS解析网址的功能没了？

### DISCUSSION THREAD

**Discussion by zhbityangjian** [vernesong/OpenClash#3276] [General]
'oa.xxx.edu.cn': '10.0.15.11'
'+.xxx.com': '10.0.15.11'
'+.xxx.edu.cn': '10.0.15.11'

每行请只填写一个规则，列表中的域名将使用指定的 DNS 进行查询，但 Fake-IP 模式下仍将返回保留地址
新版的这个功能在哪里找到？我的校园网有些网站需要内网DNS解析的

**Comment by Plutonium141** [comment]
覆写设置→dns设置下的 nameserver-policy
